### PR TITLE
Update tracker-stores.rst

### PR DESCRIPTION
**Pull request by Raghibshams456**
As suggested replace tablename with  table_name

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @tttthomasssss will take a look at it as soon as possible ✨

**Comment by Raghibshams456** [comment]
hey ..do I have to try it again??..please do let me know If I had done something wrong..

File: docs/api/tracker-stores.rst [hunk 1]
```diff
@@ -320,7 +320,8 @@ DynamoTrackerStore
                 rasa run -m models --endpoints endpoints.yml
 
 :Parameters:
-    - ``tablename`` (default: ``states``): name of the DynamoDB table
+
```
Comment (erohmensing) [inline docs/api/tracker-stores.rst:323]:
```suggestion
```

**Comment by m-vdb** [comment]
we now have a new documentation system, would it be possible to do the changes in [`docs/docs/tracker-stores.mdx`](https://github.com/RasaHQ/rasa/blob/master/docs/docs/tracker-stores.mdx) instead? (this is why there are conflicts)

**Comment by tmbo** [comment]
I'll close this due to inactivity, change has been applied to master.

**Pull request closed without merging**